cluster/addons/rbac/cluster-autoscaler/cluster-autoscaler-rbac.yaml

```diff
@@ -13,6 +13,14 @@ rules:
     resources: ["leases"]
     resourceNames: ["cluster-autoscaler"]
     verbs: ["get", "update", "patch", "delete"]
+  # TODO: remove N releases after upgrading cluster autoscaler to a version that uses leases
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["create"]
```

Comment:
it looks like this was removed in https://github.com/kubernetes/kubernetes/pull/89089 with the update to 1.17?